compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -3815,27 +3825,37 @@ public BLangNode transform(MatchStatementNode matchStatementNode) {
         return matchStatement;
     }
 
-    private BLangMatchPattern transformMatchPattern(Node matchPattern, DiagnosticPos matchPatternPos) {
+    private BLangMatchPattern transformMatchPattern(Node matchPattern) {
```

Comment:
Shall we refactor this method with an if ladder? since each clause do a return?

Reply:
Done